custom/enikshay/tasks.py

```diff
@@ -397,6 +397,14 @@ def _get_fulfilled_vouchers(self):
         ]
         return sorted(relevant_vouchers, key=self._get_fulfilled_voucher_date)
 
+    def _get_fulfilled_available_vouchers(self):
+        relevant_vouchers = [
+            voucher for voucher in self._get_all_vouchers()
+            if (voucher.get_case_property('voucher_type') == 'prescription'
+                and voucher.get_case_property('state') in ['fulfilled', 'available'])
```

Comment:
Flagging that there are other voucher states that may be applicable here, such as `paid` and `rejected`.  It kinda feels like we want a boolean version of `state`, something like `is_approved`.